package.json

```diff
@@ -31,24 +31,25 @@
     "node": ">= 12"
   },
   "devDependencies": {
```

Comment:
Can you bump the `version` (line 4) from 2 to 3 as it's a breaking change? (sorry can't comment on that line specifically)

Reply:
Can you also update engines to 14 and pin node with volta@14?